presto-main/src/main/java/com/facebook/presto/sql/planner/CompilerConfig.java

```diff
@@ -23,6 +24,8 @@
 public class CompilerConfig
 {
     private int expressionCacheSize = 10_000;
+    private int maxNodeNumInLogicalPlan = 100_000;
```

Comment:
maybe call this `maxPlanNodes` or `planNodesLimit` (and similarly for all the getters/setters, property names, etc). 

in general for variable names we try to:
1. avoid abbreviations 
2. keep them concise but descriptive

Reply:
Done